lib/modules/manager/bazelisk/extract.ts

```diff
@@ -0,0 +1,12 @@
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile {
+  const dep: PackageDependency = {
+    depName: 'bazel',
+    currentValue: content.trim().split('\n', 2)[0],
```

Comment:
```suggestion
    currentValue: content.trim().split('\n', 2)[0]?.trim(),
```

This might be needed if there's whitespace at the end of the first line

Reply:
Note the `?` shouldn't be necessary because `.split()` will always return an array of at least one entry.